### PR TITLE
LPS-52961 add modules/util/java-sass-compiler project

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
ci:retest

**Comment by gamerson** [comment]
ci:retest

**Comment by gamerson** [comment]
hey @brianchandotcom  all of our tests are passing on our OSes here in the office
linux64 - ubuntu14
linux64 - fedora 2x
linux32 - ubuntu10
linux64 - ubuntu10
win32 - winXP
win32 - windows7
win64 - windows7
mac64 - OS 10.7.x

This should be good to merge from our end.

**Comment by brianchandotcom** [comment]
hey Greg,

Can you resend:

1.) The tests exist somewhere in core, which are extracted from a binary file. We should keep the same logic in build.xml so we know exactly where they came from. In fact, they don't even need to be checked in. You can .gitignore them.

2.) The java files that are generated from an outside place shouldn't just be copied and checked in. See modules/third-party/com-ctc-wstx to see how we use maven to download it and expand it.

Basically, whatever we are manually downloading and checking in, we need to document that process via a script.

Thx Greg!
